src/BTConnectionManager.cpp

```diff
@@ -53,8 +53,12 @@ class BTConnectionManager::Private {
     CommandQueue* commandQueue = nullptr;
 
     QBluetoothDeviceDiscoveryAgent* deviceDiscoveryAgent = nullptr;
-    bool discoveryRunning = false;
+    QBluetoothLocalDevice* localDevice = nullptr;
+    int localBTDeviceState = 0;
+
+    bool discoveryRunning;
 
+    bool discoveryRunning = false;
```

Comment:
This seems sort of not right...

Reply:
Should not be such change in the patch. Probably rebase/merge issue that this shows as diff.  
The code in master contains seems to contain that  https://github.com/MasterTailer/DIGITAiL/blob/master/src/BTConnectionManager.cpp#L56

Please correct me if I'm wrong. 
(will see that after clean repost/repost of PR).